File: rnmodules/react-native-drop-view/android/src/main/java/com/reactnativedropview/DropViewViewManager.java
package com.reactnativedropview;

import android.graphics.Color;

import androidx.annotation.Nullable;

import com.facebook.react.module.annotations.ReactModule;
import com.facebook.react.uimanager.ThemedReactContext;
import com.facebook.react.uimanager.annotations.ReactProp;

@ReactModule(name = DropViewViewManager.NAME)
public class DropViewViewManager extends DropViewViewManagerSpec<DropViewView> {

  public static final String NAME = "DropViewView";

  @Override
  public String getName() {
    return NAME;
  }

  @Override
  public DropViewView createViewInstance(ThemedReactContext context) {
    return new DropViewView(context);
  }
}

File: rnmodules/react-native-drop-view/android/src/main/java/com/reactnativedropview/DropViewViewManager.kt
package com.reactnativedropview

import com.facebook.react.module.annotations.ReactModule
import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.uimanager.ThemedReactContext
import com.facebook.react.uimanager.annotations.ReactProp

@ReactModule(name = DropViewViewManager.NAME)
class DropViewViewManager :
        DropViewViewManagerSpec<DropViewView>() {
    override fun getName(): String {
        return NAME
    }

    public override fun createViewInstance(context: ThemedReactContext): DropViewView {
        return DropViewView(context)
    }

    companion object {
        const val NAME = "DropViewView"
    }
}

